app/src/main/java/fi/helsinki/opintoni/integration/leiki/LeikiRestClient.java

```diff
@@ -97,6 +100,8 @@ public LeikiRestClient(AppConfiguration appConfiguration, RestTemplate restTempl
             .queryParam("uid", recommendationUidPrefix + studentNumber)
             .queryParam("showtags", "true")
             .queryParam("unreadonly", "true")
+            .queryParam("startdate", today.toString() + "T00:00:00+0300")
```

Comment:
Olisi siistimpää luoda oikeaan aikaan asetettu `LocalDateTime` ja `.format`oida se `DateTimeFormatter`in avulla halutunlaiseksi stringiksi.